reef-bridge-project/reef-bridge-java/src/main/java/org/apache/reef/javabridge/generic/Launch.java

```diff
@@ -169,7 +169,6 @@ public static void main(final String[] args) {
         client.waitForCompletion(0);
       }
 
```

Comment:
This seems like a pure formatting change. Please remove it from the PR to make the commit as small as possible.